linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_syslog_plugin_activated/rule.yml

```diff
@@ -45,6 +45,7 @@ references:
     cobit5: APO11.04,APO12.06,BAI03.05,BAI08.02,DSS02.02,DSS02.04,DSS02.07,DSS03.01,DSS05.04,DSS05.07,MEA02.01
     iso27001-2013: A.12.4.1,A.12.4.2,A.12.4.3,A.12.4.4,A.12.7.1,A.16.1.4,A.16.1.5,A.16.1.7
     cis-csc: 1,11,12,13,14,15,16,19,3,4,5,6,7,8
+    srg: SRG-OS-000037-GPOS-00015,SRG-OS-000038-GPOS-00016,SRG-OS-000039-GPOS-00017,SRG-OS-000040-GPOS-00018,SRG-OS-000041-GPOS-00019,SRG-OS-000062-GPOS-00031
```

Comment:
Nack on:
- ``SRG-OS-000037-GPOS-00015`` because enabling this does not affect audit records having information to identify what type of event occurred 
- ``SRG-OS-000038-GPOS-00016`` because this setting does not affect audit records having date and time information
- ``SRG-OS-000039-GPOS-00017`` because this setting does not affect audit records containing information to establish where the events occurred
- ``SRG-OS-000038-GPOS-00018`` because this setting does not affect audit records containing enough information to establish the source of the events
- ``SRG-OS-000041-GPOS-00019``, because this setting does not affect audit records containing the outcome of the events
- ``SRG-OS-000062-GPOS-00031`` because this setting does not define which events get audited, ensuring DoD-defined auditable events are captured

Reply:
Changing SRG-OS-000037-GPOS-00015,SRG-OS-000038-GPOS-00016,SRG-OS-000039-GPOS-00017,SRG-OS-000040-GPOS-00018,SRG-OS-000041-GPOS-00019,SRG-OS-000062-GPOS-00031 to SRG-OS-000051-GPOS-00024.  Having audit configured to use syslog will help with review and anlysis of logs and events.

If there is a better SRG option, please let me know.

Reply:
Added SRG-OS-000342-GPOS-00133